zerver/webhooks/appfollow/view.py

```diff
@@ -15,15 +15,17 @@
 @has_request_variables
 def api_appfollow_webhook(request: HttpRequest, user_profile: UserProfile,
                           stream: Text=REQ(default="appfollow"),
-                          topic: Optional[Text]=REQ(default=None),
+                          topic: Optional[Text]=REQ(default=None, type=str),
```

Comment:
So, with `REQ` with no special arguments, the type should always be `str`.  Would it be better to adjust `REQ`, rather than changing every call point to add `type=str`?